packages/qa/tests/features/studyUpdateTests/studyUpdateWithCpmsTest.e2e.ts

```diff
@@ -57,11 +169,24 @@ test.describe('Update study and save changes locally in SE @se_184', () => {
       await expect(cpmsPage.locator('.breadcrumbs')).toContainText(`CPMS ID: ${id}`)
     })
 
+    await test.step(`And I see my expected study details in CPMS`, async () => {
+      await expect(cpmsPage.locator('#CurrentStudyRecord_UkRecruitmentSampleSize')).toHaveValue(uniqueTarget)
+      await expect(cpmsPage.locator('#CurrentStudyRecord_PlannedRecruitmentStartDate')).toHaveValue(
+        '12/06/2025 00:00:00'
+      )
+      // await expect(cpmsPage.locator('#CurrentStudyRecord_ActualOpeningDate')).toHaveValue('12/06/2024 00:00:00')
```

Comment:
Minor - are these comments needed?

Reply:
yeah these are work in progress will address in next PR